app/helpers/date-format.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export function dateFormat(value, options) {
+  if (!options.format) {
+    throw Error('moment-format is expecting format to be defined.');
+  }
+  if (value.length !== 1) {
+    throw Error('moment-format is expecting one value.');
+  }
+
+  let momentValue = moment(value[0]);
+
+  if (!momentValue.isValid()) {
+    throw Error('moment-format is expecting valid Date.');
+  }
+
+  return momentValue.format(options.format);
+}
```

Comment:
This helper makes me feel like an idiot because I think I have 2 or 3 separate helpers in another project, one for each format I want to display rather than writing a single one like this and passing in a format string.  I love this helper, nice work!